examples/es-example.js

```diff
@@ -6,13 +6,14 @@ const MKR_FISH = '0x2dfcedcb401557354d0cf174876ab17bfd6f4efd';
 const PRICE_FEED_ETH = '0xa5aA4e07F5255E14F02B385b1f04b35cC50bdb66';
```

Comment:
maybe instead of modifying a single example, you could have some tests, and thus verify that both http and ws still work... :)